quickwit/quickwit-serve/src/metrics.rs

```diff
@@ -34,6 +35,14 @@ impl Default for RestMetrics {
                 &[],
                 ["method", "status_code"],
             ),
+            request_duration_seconds: new_histogram_vec(
+                "request_duration_seconds",
+                "Response time in millisecs",
```

Comment:
we use `secs`, not seconds, elsewhere.
the comment gives the wrong unit too

Reply:
good point. Yeah this is tricky. We used secs in all place but one (apart from here) and prometheus docs recommends seconds

Reply:
Ill switch to secs.